client/src/components/profile-forms/EditProfile.js

```diff
@@ -4,48 +4,41 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { createProfile, getCurrentProfile } from '../../actions/profile';
 
+const initialState = {
```

Comment:
initial state outside of component to avoid triggering a re-render in useEffect.